projects/objects/buildings/protos/LargeResidentialTower.proto

```diff
@@ -34,38 +34,40 @@ PROTO LargeResidentialTower [
         surroundingWall IS groundSurroundingWall
```

Comment:
This PROTO appears to be broken, only 1 floor was converted. The issue might be on the hidden sub protos LargeResidentialTowerFloor or roof. 

![a](https://user-images.githubusercontent.com/44834743/132009498-5dd901b5-ed99-420c-b676-00d6041427bd.png)

Reply:
Also, I'm still getting a warning "skipped unknown rotation" from this PROTO

Reply:
This is a terrible oversight, thank you! Fixed